CONTRIBUTING.md

```diff
@@ -25,6 +25,8 @@ again.
    use your corporate email address here, not your personal address.
 2. Fork the repository into your own Github account.
 3. Please include unit tests (and integration tests if applicable) for all new code.
-4. Make sure all existing tests pass. (./gradlew goJF build integrationTest)
+4. Make sure all existing tests pass. 
+  a. In `jib-core`, run `./gradlew clean goJF build integrationTest`
+  b. In `jib-maven-plugin`, run `./mvnw clean verify`
```

Comment:
users can run fmt:format to format the maven code?

Reply:
I actually attached fmt:format to run during the validate lifecycle, which now that I think about it, is probably not a good idea because Kokoro won't pick up formatting issues.